manifests/charts/ztunnel/values.yaml

```diff
@@ -43,13 +43,7 @@ multiCluster:
   # with Istiod configuration.
   clusterName: ""
 
-# meshConfig defines runtime configuration of components.
-# For ztunnel, only defaultConfig is used, but this is nested under `meshConfig` for consistency with other
-# components.
-# TODO: https://github.com/istio/istio/issues/43248
-meshConfig:
-  defaultConfig:
-    proxyMetadata: {}
-
-# Ambient redirection mode: "iptables" or "ebpf"
-redirectMode: "iptables"
+# For ztunnel, only defaultConfig in meshConfig is used. To avoid confusion from using the full meshConfig, we define a
+# proxyConfig here to replace the meshConfig.defaultConfig
```

Comment:
this comments does not make sense, as https://github.com/istio/istio/issues/43673 trustdomain may be needed